src/ln/msgs.rs

```diff
@@ -382,7 +382,81 @@ impl NetAddress {
 	}
 }
 
-#[derive(Clone)]
+impl Writeable for NetAddress {
```

Comment:
Can you adapt the UnsignedNodeAnnouncement write/read to use these as well?

Reply:
Updated